### PR TITLE
Woods edits

### PR DESCRIPTION
**Pull request by dougwoods3**

### CODE REVIEW

**Comment by albeanth** [comment]
damn! someones been busy haha

have you tried using Jekyll?

File: users/woodsdou/index.md [hunk 1]
```diff
@@ -3,64 +3,58 @@ title: Doug-personal
...
+<img src="https://github.com/rtrp/osu-transport/blob/woods-edits/users/woodsdou/images/AdamsDiff2DwMeshBlue.png" width="450">
 
-## Current projects
+With mesh refinement and/or increasing the finite element order, we see solutions converging to the analytic solution. The data points on the following image[^1] show errors between the DGFEM transport solution and the analytic MMS solution. Lines connect data points calculated using the same finite element order.
 
-* Dissertation
-  * Currently doing literature review and collecting material for writing my dissertation proposal
+<img src="https://github.com/rtrp/osu-transport/blob/woods-edits/users/woodsdou/images/plotConvergenceRates_15.png" width="400">
 
+Diffusion limit calbulations can exhibit unphyiscial oscillations in the boundadry layer solution. If the solution is near zero, the oscillations will cause the solution to drop below zero as seen by white space in the image below[^1]. In the context of thermal radiation transport, negative fluxes mean negative temperatures, which can lead to the equations of state to calculate negative densities and pressures.
```
Comment (albeanth) [inline users/woodsdou/index.md]: misspelled "calculations" and "unphysical" and "boundary"

haha someone didn't proof read 😛

**Comment by albeanth** [comment]
@dougwoods3 I made a comment in the "files changed tab" you have a few misspellings you should fix.

File: users/woodsdou/index.md (hunk 1 above)
Comment (albeanth) [inline users/woodsdou/index.md]: "negative fluxes mean negative temperatures, which can lead to the equations of state to calculate negative densities and pressures."

that sounds... 💩 

....yes I'm getting sick of reading papers.

**Comment by dougwoods3** [comment]
@albeanth Have not tried Jekyll yet. Seen it around the interwebs but not sure what it is yet.

File: users/woodsdou/index.md [hunk 2]
```diff
@@ -3,64 +3,58 @@ title: Doug-personal
...
 
+Diffusion limit calculations can exhibit unphyisical oscillations in the boundary layer solution. If the solution is near zero, the oscillations will cause the solution to drop below zero as seen by white space in the image below[^1]. In the context of thermal radiation transport, negative fluxes mean negative temperatures, which can lead to the equations of state to calculate negative densities and pressures.
 
-* Intern at Idaho National Laboratory
-  * I work for Daniel Schwen and Dan Wachs on a variety of projects
-  * Some of this includes [MOOSE](http://mooseframework.org) development, modeling and simulation
+<img src="https://github.com/rtrp/osu-transport/blob/woods-edits/users/woodsdou/images/TP1Log.png" width="500">
 
-***
+These oscillatios can also be seen in problems with varying material opacities. The image below[^1] is the solution to a multi-material problem with opacities ranging several orders of magnitude, similar to thermal radiation transport problems of practical interest. Oscillations can be seen in various regions of the problem.
```
Comment (albeanth) [inline users/woodsdou/index.md]: "oscillations"

Comment (dougwoods3) [inline users/woodsdou/index.md]: fixed 1 spelling.

File: users/woodsdou/index.md (hunk 1 above)
Comment (dougwoods3) [inline users/woodsdou/index.md]: fixed 3 spellings.

**Comment by albeanth** [comment]
@dougwoods3 it's a way you can see your changes on the webpage before pushing commits. So if you want to see how your images look before merging them into gh-pages you can. Jackson helped me get it set up last night. If you want, I can try and get you set up too.

**Comment by dougwoods3** [comment]
@albeanth that sounds practical.

**Comment by albeanth** [comment]
@dougwoods3 feel free to merge in your changes. If it looks like 💩  we can undo the merge.

**Comment by harterj** [comment]
:shipit:

**Pull request merged**